sc/src/main/webapp/WEB-INF/jsp/tiles/views/experiment-page/experiment-description.jsp

```diff
@@ -32,7 +33,7 @@
             </h3>
 
             <h5>${type}</h5>
-
+            <div>Number of cells: ${numberOfCells}</div>
```

Comment:
Use `<fmt:formatNumber value="${numberOfCells}"/>`, you’ll need to add `<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>` at the top.